astropy/io/votable/tests/converter_test.py

```diff
@@ -146,6 +146,41 @@ def test_float_mask_permissive():
     assert c.parse('null', config=config) == (c.null, True)
 
 
+def test_double_array():
+    config = {'verify': 'exception'}
+    field = tree.Field(None, name='c', datatype='double', arraysize='3',
+                       config=config)
+    data = (1.0, 2.0, 3.0)
+    c = converters.get_converter(field, config=config)
+    assert c.output(1.0, False) == '1'
+    assert c.output(1.0, [False, False]) == '1 1'
+    assert c.output(data, False) == '1 2 3'
+    assert c.output(data, [False, False, False]) == '1 2 3'
+    assert c.output(data, [False, False, True]) == '1 2 NaN'
+
+    with pytest.raises(ValueError, match='shape mismatch'):
```

Comment:
I think it is okay to raise a standard Python exception here. I can't think of anything that would cause this to actually happen in the real world? And if it does, then someone should fix it so doesn't really matter what type of exception is raised.